README.md

```diff
@@ -1,18 +1,50 @@
 # vets-api-clients
 
-This repository contains resources for accessing vets-api as a third-party
-client. It includes API specifications, sample clients, and general
-access information.
+⚠️ **Materials in this repo may be outdated as of July 2019.** This repository contains resources for accessing vets-api as a third-party client. It includes API specifications, sample clients, and general access information.   Anything older than April 2019 may be considered deprecated.  **For the most up-to-date API documentation, please visit [developer.va.gov/explore](https://developer.va.gov/explore).**
```

Comment:
IMO this README should not mention `vets-api`. That's a specific application that implements a subset of the APIs on our platform. This repo contains resources for accessing all of the APIs on the platform, without regard for which application implements them.

Reply:
@dvogel ok thanks - how would you reword this?  I was trying to include the original wording but am not clear on the distinctions between `vets-api`, `vets-api-clients` and `developer-portal`.

Reply:
I would change "vets-api" to "VA APIs". For example, I would change this:

> accessing vets-api as a third-party client

to:

> accessing VA APIs.

Reply:
Done, thanks.